pkg/operation/botanist/addons.go

```diff
@@ -285,7 +254,7 @@ func (b *Botanist) generateOptionalAddonsChart(_ context.Context) (*chartrendere
 
 	return b.ShootClientSet.ChartRenderer().Render(filepath.Join(charts.Path, "shoot-addons"), "addons", metav1.NamespaceSystem, map[string]interface{}{
 		"global":               global,
-		"kubernetes-dashboard": kubernetesDashboard,
+		"kubernetes-dashboard": common.GenerateAddonConfig(nil, kubernetesDashboardEnabled),
```

Comment:
Nit: you can get rid of this short lived variable `kubernetesDashboardEnabled`

Reply:
I prefer keeping it as a variable.